apps/server/src/modules/files-storage-client/files-storage-client.module.ts

```diff
@@ -3,11 +3,12 @@ import { LoggerModule } from '@core/logger';
 // The files-storage-client should not know the copy-helper
 import { CopyHelperModule } from '@modules/copy-helper/copy-helper.module';
 import { CqrsModule } from '@nestjs/cqrs';
+import { AmqpConnection } from '@golevelup/nestjs-rabbitmq';
 import { CopyFilesService, FilesStorageClientAdapterService, FilesStorageProducer } from './service';
 
 @Module({
 	imports: [LoggerModule, CopyHelperModule, CqrsModule],
-	providers: [FilesStorageClientAdapterService, CopyFilesService, FilesStorageProducer],
+	providers: [FilesStorageClientAdapterService, CopyFilesService, FilesStorageProducer, AmqpConnection],
```

Comment:
Warum Brauchen wir das hier?

Reply:
Sobald man das Module einzeln irgendwo injected wo Ampq fehlt, kann Nest das nicht mehr auflösen. 
Mir ist das aufgefallen, da ich ein modules welches das hier verwendet in feather services test injected habe.

Ich schätze das golevelup den nest ampq services so präpariert hat das er als injectable durch geht.
Ich schätze das wir ihn im Server irgendwo anders bereits hinzugefügt hatten und es daher dort lief.

Reply:
Ich bin mir nicht sicher ob das die selbe instance dann ist, die über RabbitMQModule rein kommt. Können wir das in feather injecten

Reply:
done